### PR TITLE
Refactor for `managedlist` `ngListFilterable` and `glossesSelectorForTextualWitness`

### PR DESCRIPTION
**Pull request by thehabes**
Uses HTML Elements instead of Template Literals.  This is in reaction to the improvements and bug fixes this refactor helped with in `filterableListItem`.  Now all the "Glosses List" and "Glosses List Item" templates are in the same style and contain the same improvements.

### CODE REVIEW

File: js/gloss-witness.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ function setWitnessFormDefaults(){
     form.$isDirty = true
     form.querySelector("input[deer-key='creator']").removeAttribute("deer-source")
     // For when we test
-    //form.querySelector("input[deer-key='creator']").value = "BryanManageGlosses"
+    // form.querySelector("input[deer-key='creator']").value = "BryanRefactor"
```
Comment (cubap) [inline js/gloss-witness.js:34]: funny

File: js/manageGlossModal.js [hunk 2]
```diff
@@ -85,7 +85,7 @@ class ManageGlossModal extends HTMLElement {
             const glossTitle = `${published ? "✓" : "❌"}  ${glossData.title}`
 
             const removeBtn = `<input type="button" value="delete" glossid="${glossID}" data-type="named-gloss" class="removeCollectionItem button error is-small" title="Delete This Entry">`
-            const visibilityBtn = `<input type="button" value="${published ? "unpublish" : "publish"}" class="togglePublic button ${published ? "error" : "success"} is-small" glossid="${glossID}" title="Toggle public visibility"/>`
+            const visibilityBtn = `<input type="button" value="${published ? "unpublish" : "publish"}" class="togglePublic button ${published ? "error" : "success"} is-small" glossid="${glossID.replace(/^https?:/, 'http:')}" title="Toggle public visibility"/>`
```
Comment (cubap) [inline js/manageGlossModal.js:88]: This trick should be extracted soon, if it is needed like this. It may be in utils already

**Review by cubap — APPROVED** [review]
I have no reason to hold this back

File: js/manageGlossModal.js (hunk 2 above)
Comment (thehabes) [reply js/manageGlossModal.js:88]: There is some real funny stuff going on with manage glosses.  We need to go through this again.  See #179

**Pull request merged**